content/2-running-automation-testing/guide.md

```diff
@@ -0,0 +1,202 @@
+# 2. Running automation test with Kobiton
```

Comment:
Use `kobiton-org-demo` account, @baotn166 please transfer the credential to Lily